docs/general-usage-instructions.html

```diff
@@ -0,0 +1,21 @@
+<!--Installation and configuration instructions. HTML tags are allowed.-->
+
+<ol>
+  <li>Install the plugin from <b>Marketplace</b></li>
+  <li>
+    Open Git Machete tab:
+    <ul>
+      <li><b>Git</b> (tool window) -> <b>Git Machete</b> (tab)</li>
+      <li>Use key shortcut: Ctrl + Alt + Shift + M</li>
+    </ul>
+  <li>From the left side plugin's toolbar select <b>Open Machete File</b></li>
+  <li>Fill the file with local branch names (the completion may be useful)</li>
+  <li>
+    Refresh the graph:
+    <ul>
+      <li>save file and click <b>Refresh Status</b> (Ctrl + F5) button from the left side plugin's toolbar</li>
+      <li><b>Load Machete Changes</b> (Ctrl + Shift + O)</li>
+    </ul>
+  </li>
+  <li>Start to use it! :)</li>
```

Comment:
Maybe suggest using the rebase action from left-side toolbar and/or seeing what actions are available for each branch in context menu